src/on-chain-value.ts

```diff
@@ -27,48 +25,37 @@ export const chainValueDataPoint: APIGatewayProxyHandler = async (event): Promis
     };
   }
 
-  const provider = config.providers[chainId];
-  if (!provider) {
+  const db = await initDb();
+  if(db === undefined) {
     return {
       statusCode: 500,
       headers: config.headers,
-      body: makeError("We don't have a provider for that chainId"),
+      body: makeError("An error occurred while trying to initialize the database"),
     };
   }
 
-  const dapiServerAddress = contracts.DapiServer[chainId];
-  if (!dapiServerAddress) {
-    return {
-      statusCode: 500,
-      headers: config.headers,
-      body: makeError("We don't have a dapiServer for that chainId"),
-    };
-  }
-
-  const voidSigner = new ethers.VoidSigner(
-    ethers.constants.AddressZero,
-    new ethers.providers.JsonRpcProvider(provider)
+  const operation = async () => db.query(`
```

Comment:
It's good that this is now working, well done, but this query is very bad because it's subject to a SQL injection attack.
You need to use a parameterised version of this query.

Reply:
Just fixed in [this commit](https://github.com/api3dao/beacons-api/pull/5/commits/c0309ce4e7f5c353da5dc6060db192b5bdf3ae9e).